### DISCUSSION TITLE
How is Livewire 3 called across components? - $5

### DISCUSSION THREAD

**Discussion by Aidi225** [livewire/livewire#7650] [1. Help] [answered]
I posted this question on 1024 and paid $5 as a bounty.

[Livewire 3: How to Make Cross-Component Calls? - 1024.dev](https://1024.dev/questions/3050360370/livewire-3-how-to-make-cross-component-calls)

**Comment by danie-ramdhani** [comment, marked as answer]
maybe you're looking for https://livewire.laravel.com/docs/events#listening-for-events ?

example:
https://wirebox.app/b/4q232

**Comment by Aidi225** [comment]
[https://1024.dev/questions/3050360370/livewire-3-how-to-make-cross-component-calls#answer-2737066152](https://1024.dev/questions/3050360370/livewire-3-how-to-make-cross-component-calls#answer-2737066152)

[https://1024.dev/questions/3050360370/livewire-3-how-to-make-cross-component-calls#answer-2583304582](https://1024.dev/questions/3050360370/livewire-3-how-to-make-cross-component-calls#answer-2583304582)
